tests/ui/test_signposting_ui.py

```diff
@@ -11,23 +11,58 @@
 """
 
 
-def test_link_in_landing_page_response_headers(running_app, client, record):
-    res = client.head(f"/records/{record.id}")
+def test_link_in_landing_page_response_headers(running_app, client, record_with_file):
+    ui_url = f"https://127.0.0.1:5000/records/{record_with_file.id}"
+    api_url = f"https://127.0.0.1:5000/api/records/{record_with_file.id}"
+    filename = "article.txt"
+
+    res = client.head(f"/records/{record_with_file.id}")
```

Comment:
question/comment: I think the `HEAD` implementation for Flask/Invenio is that we just treat it as a GET request and skip the body of the response. In that case, we're not saving anything in terms of computation/performance (if that was the original goal of just testing the HEAD response only).

IMHO, it's ok to keep as is, since none of the logic done for generating the header links is that much more complex or adds that big of an overhead compared to the rest of the GET response.

Reply:
The `Link` header should be included in both `GET` and `HEAD`, as stated in the [FAIR Signposting docs](https://signposting.org/FAIR/#typedlinks) says:
> In addition to being available via HTTP GET requests, the HTTP header that contains Link is accessible via the HTTP HEAD request, which only returns transaction metadata not a resource representation. As such machine agents can obtain a map for their journey by issuing a HTTP HEAD even against resources that have access restrictions. All the while saving bandwidth and hence energy.

- [ ] Modify the tests to assert not only `HEAD`, but also `GET`.